.chronus/changes/continuation_token-2025-1-18-16-47-36.md

```diff
@@ -0,0 +1,7 @@
+---
+changeKind: feature
+packages:
+  - "@azure-tools/typespec-client-generator-core"
+---
+
+Add `SdkPagingServiceMetadata` type to store all paging related info.
```

Comment:
do we have a weird char here inside the back quote?
![image](https://github.com/user-attachments/assets/fe18db4d-dec5-48ff-97f5-fb2ef55c3384)